wagtail/images/models.py

```diff
@@ -403,16 +404,23 @@ def run(self, image, output):
                 # Developer specified an output format
                 output_format = env['output-format']
             else:
-                # Default to outputting in original format
-                output_format = original_format
-
-                # Convert BMP files to PNG
-                if original_format == 'bmp':
-                    output_format = 'png'
+                # Convert bmp and webp to png by default
+                default_conversions = {
+                    'bmp': 'png',
+                    'webp': 'png',
```

Comment:
This line feels a bit odd as WEBP is supposed to replace PNG (some day). Maybe we should document the fact that this currently happens and show an example of how to configure Wagtail so that user uploaded WEBP files remain as WEBP

Reply:
@kaedroho Isn't that already covered by https://github.com/wagtail/wagtail/pull/5249/commits/5947f9cf38aa757c6cd14129ecde9ac142945673#diff-9909cf0a580b7bd0cfd1756593179301R29 ?